src/screens/Voucher/VoucherHomeScreen.tsx

```diff
@@ -37,11 +50,41 @@ export const VoucherHomeScreen = ({ navigation, route }: StackScreenProps<any>)
   // refresh if the refreshAuth param changed, which happens after login
   useEffect(refreshAuth, [route.params?.refreshAuth]);
 
+  const { mutate: mutateLogIn } = useMutation(logIn);
+
+  useEffect(() => {
+    const accountCheck = async () => {
+      const loginInfo = await voucherMemberLoginInfo();
+      const savedDate =
+        (await readFromStore(SAVED_DATE_OF_LAST_ACCOUNT_CHECK)) || moment().format('YYYY-MM-DD');
+
+      if (loginInfo && !moment().isSame(savedDate, 'day')) {
+        mutateLogIn(JSON.parse(loginInfo as string) as VoucherLogin, {
+          onSuccess: (responseData) => {
+            if (!responseData?.member) {
+              storeVoucherAuthToken();
+              storeVoucherMemberId();
+              storeVoucherMemberLoginInfo();
+              refresh();
+            }
+          }
+        });
+
+        addToStore(SAVED_DATE_OF_LAST_ACCOUNT_CHECK, moment().format('YYYY-MM-DD'));
+        setLoadingAccountCheck(false);
```

Comment:
it is still here and can be removed, as it will be performed two lines later

Reply:
https://github.com/smart-village-solutions/smart-village-app-app/commit/9153072e8eabb29280e8c4ec0f276b3ca404583d